### PR TITLE
Add management command to inspect max-val of id column for db tables

### PR DESCRIPTION
**Pull request by sravfeyn**
https://dimagi-dev.atlassian.net/browse/IIO-904
@snopoke

### CODE REVIEW

**Review by snopoke — COMMENTED** [review]
Thanks for this @sravfeyn. I had originally thought that we would just list all the sequences and check how close they are to their max value. The method you have here does work but has some edge cases that aren't accounted for like primary key columns not called `id`.

**Comment by sravfeyn** [comment]
Updated.

File: corehq/sql_db/management/commands/print_max_id_of_pg_tables.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        parser.add_argument(
+            'dbname',
+            help='Django db alias'
+        )
+        parser.add_argument(
+            '--minval',
+            type=int,
+            dest='minval',
+            help='print only sequences where the last_value is more than this provided minvalue'
+        )
```
Comment (snopoke) [inline corehq/sql_db/management/commands/print_max_id_of_pg_tables.py:38]: what would you put here in practice?

you should be able to get the sequence min / max value from postgres

Comment (sravfeyn) [reply corehq/sql_db/management/commands/print_max_id_of_pg_tables.py:38]: Ok, modified to pull max_value and display it in output.

**Review by snopoke — APPROVED** [review]

**Pull request merged**